guides/manual-installation.md

```diff
@@ -0,0 +1,279 @@
+## Scope
+
+This documents covers the step to manually install trento server without relying on the trento helm chart or kubernetes all together. The lastest
+available version of SUSE Linux Enterprise Server for SAP Applications is used as the base operating system, which is 15 SP5 at the time of writing.
+
+## List of dependencies
+
+- prometheus
+- postgresql
+- rabbitmq
+- docker runtime
+
+## Installation
+
+### Install prometheus (Optional)
+
+#### Option 1: Use existing installation
+
+If you already have a prometheus server running, you can skip this step. , you can skip the inst
+
+#### Option 2: Do not use prometheus
+
+If you don't have a Prometheus instance running and you don't want to install one, you can continue without doing so. The downside is that trento will
+not be able to collect metrics from prometheus and the integrated graphs will not work.
```

Comment:
The explanation would be more helpful if we move it in line 16 under the ### Install prometheus (Optional)" header. 
As a reader i would know why i would need Prometheus and then with decide with which option to use or not use.